src/mpid/ch4/netmod/ofi/ofi_init.c

```diff
@@ -638,7 +665,7 @@ static int flush_send(int dst, int vni, MPIDI_OFI_dynamic_process_request_t * re
     int mpi_errno = MPI_SUCCESS;
 
     MPIR_Comm *comm = MPIR_Process.comm_world;
-    fi_addr_t addr = MPIDI_OFI_av_to_phys(MPIDIU_comm_rank_to_av(comm, dst), vni, vni);
+    fi_addr_t addr = MPIDI_OFI_av_to_phys(MPIDIU_comm_rank_to_av(comm, dst), vni, vni, 0);
```

Comment:
Oh, another place I missed. We need add `nic` to the parameters here and iterate over all `nic`s in `flush_send_queue`.

Reply:
We talked offline about this. This PR doesn't actually use any NIC other than NIC 0 so for now, this will not iterate over all of them. I'll expand that in the follow-on PRs where we actually use more than one NIC at a time.